enhancements/machine-api/cluster-autoscaler-integration.md

```diff
@@ -0,0 +1,449 @@
+---
+title: cluster-autoscaler-integration
+authors:
+  - "@JoelSpeed"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-06-01
+last-updated: 2020-06-01
+status: implemented
+see-also:
+  - "/enhancements/machine-api/cluster-autoscaler-operator.md"  
+replaces:
+superseded-by:
+---
+
+# Cluster Autoscaler Integration
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) is a tool that users
+can deploy to a cluster to enable the automatic provisioning of extra resources when workloads exceed the cluster's
+capacity.
+It can also be configured to scale down resources if they are surplus to current requirement.
+
+In normal operation, a user would configure the Cluster Autoscaler to talk directly to their cloud provider and create
+new instances by expanding a nodegroup, such as an Autoscaling Group on AWS EC2.
+Because OpenShift users the Machine-API to create compute resources, OpenShift clusters do not have nodegroups in the
```

Comment:
typo r/users/uses
May be "Because OpenShift uses a cloud provider agnostic machine API for compute management..."